### ISSUE TITLE
What do 10X_sample11 and 10X_sample51 refer to and how were they processed?

### ISSUE THREAD

**Issue by Baytars** [liboxun/A-single-factor-elicits-multilineage-reprogramming-of-astrocytes-in-the-adult-mouse-striatum#1]
In `0.Run_Scrublet.ipynb` there are two unknown datasets named 10X_sample11 and 10X_sample51 respectively and I was struggling to make sense of them. How they were generated from the public 10X 1.3M brain cell dataset is unclear. I wish scripts can be available showing how `/project/GCRB/Hon_lab/s418610/Projects/03.Invivo_neuronal_reprogramming/BL41-42_with_10X/1.3M_brain_11_51/aggr_10x/outs/filtered_feature_bc_matrix.h5` is generated.

**Comment by liboxun** [comment]
Thank you for pointing this out. Sorry that documentation was missing. I will add the documentation to the repository. 

For now, here's an answer: 

`/project/GCRB/Hon_lab/s418610/Projects/03.Invivo_neuronal_reprogramming/BL41-42_with_10X/1.3M_brain_11_51/aggr_10x/outs/filtered_feature_bc_matrix.h5` 

was generated by the following code using cell ranger 2.1.0: 

`$CELLRANGER aggr --id=aggr_10x`
`        --csv=/project/GCRB/Hon_lab/s418610/Projects/03.Invivo_neuronal_reprogramming/BL41-42_with_10X/1.3M_brain_11_51/aggr.csv`
`        --normalize=mapped`

and the `aggr.csv` file in the command above has the following content (5 lines): 

`library_id,molecule_h5,group`
`BL41,/project/GCRB/Hon_lab/s418610/Projects/03.Invivo_neuronal_reprogramming/BL41-42_Lenti-Dlx2_tdTomato/BL41_10x/outs/molecule_info.h5,reprogrammed`
`BL42,/project/GCRB/Hon_lab/s418610/Projects/03.Invivo_neuronal_reprogramming/BL41-42_Lenti-Dlx2_tdTomato/BL42_10x/outs/molecule_info.h5,control`
`10X_brain_sample11,/project/GCRB/Hon_lab/s418610/Data/scRNA-seq/10X_1.3M_neurons/mapping/10X_brain_sample11_10x/outs/molecule_info.h5,10X`
`10X_brain_sample51,/project/GCRB/Hon_lab/s418610/Data/scRNA-seq/10X_1.3M_neurons/mapping/10X_brain_sample51_10x/outs/molecule_info.h5,10X`

Basically, what happened in the cell ranger command above was I used the `cell ranger aggr` function to aggregate the 4 `.h5` files previously generated from fastq files by the cell ranger alignment pipeline for the 4 libraries below, in that order: 
* BL41 (cells from the reprogramming mouse group)
* BL42 (cells from the negative control mouse group)
* 10X 1.3M brain cells sample 11
* 10X 1.3M brain cells sample 51

That's why in `0.Run_Scrublet.ipynb` I renamed these 4 libraries accordingly. 

Hope this helps! Sorry this came from when I was really bad at documenting my code.

**Comment by Baytars** [comment]
Thanks for the information. Since this process is time-consuming, could you please provide with the `filtered_feature_bc_matrix.h5` file to make reproduction easier?

**Comment by liboxun** [comment]
The file has been added to `/Data`. Please let me know if you can use it successfully for `0.Run_Scrublet.ipynb`.

**Comment by liboxun** [comment]
Closed due to inactivity.
